docs/IE_DG/Integrate_with_customer_application_new_API.md

```diff
@@ -36,7 +36,7 @@ InferenceEngine::Core core;
 ```cpp
 auto network = core.ReadNetwork("Model.xml");
 ```
-**Or read the model from ONNX format** (.onnx and .prototxt are supported formats)
+**Or read the model from ONNX format** (.onnx and .prototxt are supported formats). More information about ONNX format support is [here](./ONNX_Supported_Ops.md).
```

Comment:
You can find more information about the ONNX format support in ... (put a title of the referenced topic as a link text).

Reply:
Fixed